app/routes/circulars/circulars.server.ts

```diff
@@ -591,7 +591,9 @@ export async function approveChangeRequest(
   if (changeRequest.zendeskTicketId)
     promises.push(closeZendeskTicket(changeRequest.zendeskTicketId))
 
-  await Promise.all(promises)
+  if (newVersionNumber) {
```

Comment:
Remove this `if` statement. `autoincrementVersion.put` will always return an integer greater than zero, so the conditional branch is always taken.

If `autoincrementVersion.put` fails for any reason, it will raise an exception and halt execution of the function, won't it?